deployment/construct.yaml

```diff
@@ -0,0 +1,28 @@
+version: 0.0.2
+name: micro-sam
+installer_type: pkg  #[osx]  # This will trigger pkg build on Mac Os. On windows and linux, native build will be done and this has no effect.
+welcome_image: ./box_prompt.png
+header_image: ./box_prompt.png
+icon_image: ./box_prompt.png
+license_file: ../LICENSE
+channels:
+  - pytorch
+  - nvidia
+  - conda-forge
+
+specs:
+  - python =3.11
+  - napari
+  - pooch
+  - python-elf
+  - pytorch
+  - pytorch-cuda>=11.7  # you may need to update the cuda version to match your system
+  - torchvision
+  - tqdm
+
+# conclusion_text: short quickstart guide
```

Comment:
Do you have suggestions for conclusion_text, welcome_text and company?

Reply:
I will think about it. For now, can you just put some dummy text like `Lorem ipsum`? So that I see where exactly this text shows up before writing it.